jcl/src/java.base/share/classes/openj9/internal/foreign/abi/InternalDowncallHandler.java

```diff
@@ -25,6 +25,8 @@
 /*[IF JAVA_SPEC_VERSION >= 22]*/
 import java.util.Arrays;
 /*[ENDIF] JAVA_SPEC_VERSION >= 22 */
+import java.util.ArrayDeque;
```

Comment:
I thought I saw this fixed before but these imports should be moved within JAVA_SPEC_VERSION >= 22.